src/Aquifer.AI/OpenAiTranslationService.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System.Text;
+using System.Text.RegularExpressions;
+using Aquifer.Common.Clients;
+using Aquifer.Common.Utilities;
+using OpenAI.Chat;
+
+namespace Aquifer.AI;
+
+public interface ITranslationService
+{
+    /// <summary>
+    /// Translates plain text.
+    /// Note: It's up to the caller to guarantee that the text length is not too long.
+    /// </summary>
+    public Task<string> TranslateTextAsync(
+        string text,
+        (string Iso6393Code, string EnglishName) destinationLanguage,
```

Comment:
Is this ISO code being used anywhere? I don't think we'd ever pass it to OpenAI.

Reply:
It's being used to look up the correct prompt for HTML.  I decided to just pass the same language information to both methods (text and HTML) for consistency even though the ISO code isn't needed for straight text translation.